### ISSUE TITLE
Upload button has a download icon

### ISSUE THREAD

**Issue by avaer** [webaverse/app#152]

**Comment by avaer** [comment]
<img width="110" alt="capture3" src="https://user-images.githubusercontent.com/6926057/96201296-3c524600-0f2a-11eb-9a18-dcd8e33def74.PNG">

**Comment by ChrisEddy** [comment]
my thought was I wanted something pointing down so symbolizes your uploading something into the below list

**Comment by avaer** [comment]
Yeah I can kind of see that... but I just see this arrow as confusing. Upload/download is an established icon pattern and I wouldn't want to deviate from it.

It's a minor issue either way.
